SpywareFilter/sections/specific.txt

```diff
@@ -18,6 +18,11 @@
 kinotan.ru##.counts
 ||fstrk.net/api/tracker/
 ||massdrop.com/*/pixelrb?
+||yastatic.net^$domain=sims4pack.ru,third-party
```

Comment:
Not spyware. Social filter already blocks share.js and script file blocked contains this:
https://github.com/es-shims/es5-shim